http/http-advanced/src/test/java/io/quarkus/ts/openshift/http/AbstractHttpTest.java

```diff
@@ -177,7 +179,8 @@ public void vertxHttpClientRedirection() throws InterruptedException, URISyntaxE
     }
 
     protected String getAppEndpoint() {
-        return appEndpoint.toString();
+        String endpoint = appEndpoint.toString();
+        return endpoint.endsWith("/") ? endpoint.substring(0, endpoint.length() - 1) : endpoint;
```

Comment:
I think that is easier to understand the other round way, that is also how the current `master` branch works. 

```
protected String getAppEndpoint() {
        String endpoint = appEndpoint.toString();
        return endpoint.endsWith("/") ? endpoint : endpoint + "/";
    }
```

And all the other changes are gone, with this approach.